domains/clyron.json

```diff
@@ -6,6 +6,10 @@
     "email": "onenonlyclyron@gmail.com"
   },
   "record": {
-    "CNAME": "theclyron.github.io"
+    "CNAME": "theclyron.github.io",
+    "MX": [
+      "mx1.improvmx.com",
+      "mx2.improvmx.com"
+    ]
```

Comment:
You cannot use CNAME with any other records. If you want to you can replace the CNAME with A records.